app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -252,9 +252,9 @@ public ChatMessage getItem(int index) {
         if (getData() != null && getData().isValid()) {
             if (index == getData().size()) {
                 if (isSusiTyping) {
-                    return new ChatMessage(-404, "", "", false, false, false, "", null, "", 0);
+                    return new ChatMessage(-404, "", "", false, false, false, "", null, "", "");
                 }
-                return new ChatMessage(-405, "", "", false, false, false, "", null, "", 0);
+                return new ChatMessage(-405, "", "", false, false, false, "", null, "", "");
```

Comment:
>What does -404 and -405 refer to here?

@betterclever these are two random IDs. They are never used. They are negative because we don't want them to clash with actual message IDs. These are used to refer empty( zero height) view holders. IMO, there is no need to define them as constants because they are never referenced again. We just wanted to put any random number, so we used these.